plugins/pastefromwordimage/plugin.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or http://ckeditor.com/license
+ */
+
+( function() {
+	'use strict';
+
+	CKEDITOR.plugins.add( 'pastefromwordimage', {
+		requires: 'pastefromword',
+		init: function() {}
+	} );
+
+	/**
+	 * Help methods used by paste from word image plugin.
+	 *
+	 * @since 4.8.0
+	 * @class CKEDITOR.plugins.pastefromwordimage
+	 */
+	CKEDITOR.plugins.pastefromwordimage = {
+		/**
+		 * Methods parse RTF clipboard to find embedded images.
+		 *
+		 * @private
+		 * @since 4.8.0
+		 * @param {String} rtfClipboard Data obtained from RTF clipboard.
+		 * @returns {Array} Contains array of objects with images or empty array if there weren't a match.
+		 * @returns {Object} return.Object Single image found in `rtfClipboard`.
+		 * @returns {String/null} return.Object.hex Hexadecimal string of image embedded in RTF clipboard.
+		 * @returns {String/null} return.Object.type String represent type of image, allowed values: 'image/png', 'image/jpeg' or `null`
+		 */
+		extractImagesFromRtf: function( rtfClipboard ) {
```

Comment:
`rtfClipboard` could be renamed to `rtfContent` it doesn't have to be anything related to clipboard. It might be just simply a plain RTF file and that's fine too.